android/app/src/main/java/app/covidshield/module/CovidShieldModule.kt

```diff
@@ -50,6 +60,20 @@ class CovidShieldModule(context: ReactApplicationContext) : ReactContextBaseJava
         }
     }
 
+    @ReactMethod
+    fun fetchExposureConfiguration(url: String, promise: Promise) {
+        promise.launch(this) {
+            val request = Request.Builder().url(url).build()
```

Comment:
When merging this code, be sure to incorporate the `cache-control` code:

`.cacheControl(CacheControl.Builder().noStore().build())`